phow-comps.md

```diff
@@ -0,0 +1,39 @@
+---
+name: "@phow-comps"
+project: "rc paper airplane"
+---
+
+# Project Name
```

Comment:
```suggestion
# rc paper airplane
```

Reply:
Hey @bellsea, I initally thought these were ready made too,  but after looking at videos online I realised this was not the case. The only thing is that they give you an instruction guide. From which you assemble the parts. Then you use the given foam boards/wooden dowels and planks to make a plane, to which you attach the assembled flying system and connect it to your plane. Then you use either an rc/phone and use bluetooth to control the plane. Im still gonna have a look at the guide you sent, but i feel like this will still be a hardware project